### PR TITLE
Close #19

### LINKED ISSUE #19: Remove input end tag
In HTML, the <input> tag has no end tag.

`<input ... />` -> `<input ... >`

### PR DESCRIPTION
**Pull request by vcpablo**
- introduce valid id formats validation
- remove input close tag

### CODE REVIEW

**Review by thiagodp — CHANGES_REQUESTED** [review]
Please provide tests along with the new code. 
Regarding the `.config.sample`, I recommend taking a look at the Slack discussion about the configuration format. I think it's better defining code for wrappers, widgets, etc., instead of CSS classes.

File: .config.sample [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (WillianGoncalves) [inline .config.sample:1]: This file is not supposed to be committed now. It will be part of another issue.

File: src/widgets/prop.ts [hunk 2]
```diff
@@ -15,3 +39,11 @@ export function formatProperties(props: any, validProperties: string[]): string
 
     return Object.keys(props).reduce(formatValid, '').trimRight()
 }
+
+export function createLabel(name: string, id: string): string {
+
```
Comment (WillianGoncalves) [inline src/widgets/prop.ts:44]: Remove empty lines.

**Review by WillianGoncalves — CHANGES_REQUESTED** [review]
Just a few changes, besides what @thiagodp said about tests.

**Review by thiagodp — APPROVED** [review]

**Pull request merged**